packages/tabs/src/styles.scss

```diff
@@ -0,0 +1,104 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/typography";
+
+.tabList {
+  border-bottom: 1px solid rgba($color-gray-600-rgb, 0.1);
+  padding: 0 $spacing-md;
```

Comment:
By default, our components shouldn't include margin or padding around the wrapper of the component. Consumer can add spacing if they need it.

Reply:
I'm totally with you (I went through all of the ones we need to remove in https://github.com/cultureamp/kaizen-design-system/issues/1894). this is a tough one though. This isn't quite "around the wrapper" because of the bottom border. Marc has a particular margin he wants here, and said that too commonly people are leaving this out.

![image](https://user-images.githubusercontent.com/1811583/140666040-d2aedc9c-3e71-4d07-b09b-5befb19a1666.png)